src/main/java/com/thevoxelbox/voxelsniper/brush/type/CopyPastaBrush.java

```diff
@@ -136,21 +136,13 @@ private void doPasta(Snipe snipe) {
 			for (int j = 0; j < this.arraySize[1]; j++) {
 				for (int k = 0; k < this.arraySize[2]; k++) {
 					int currentPosition = i + this.arraySize[0] * j + this.arraySize[0] * this.arraySize[1] * k;
-					Block block;
-					switch (this.pivot) {
-						case 180:
-							block = clampY(this.pastePoint[0] - this.offsetPoint[0] - i, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] - this.offsetPoint[2] - k);
-							break;
-						case 270:
-							block = clampY(this.pastePoint[0] + this.offsetPoint[2] + k, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] - this.offsetPoint[0] - i);
-							break;
-						case 90:
-							block = clampY(this.pastePoint[0] - this.offsetPoint[2] - k, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] + this.offsetPoint[0] + i);
-							break;
-						default: // assume no rotation
-							block = clampY(this.pastePoint[0] + this.offsetPoint[0] + i, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] + this.offsetPoint[2] + k);
-							break;
-					}
+					Block block = switch (this.pivot) {
+						case 180 -> clampY(this.pastePoint[0] - this.offsetPoint[0] - i, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] - this.offsetPoint[2] - k);
+						case 270 -> clampY(this.pastePoint[0] + this.offsetPoint[2] + k, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] - this.offsetPoint[0] - i);
+						case 90 -> clampY(this.pastePoint[0] - this.offsetPoint[2] - k, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] + this.offsetPoint[0] + i);
+						default -> // assume no rotation
+							clampY(this.pastePoint[0] + this.offsetPoint[0] + i, this.pastePoint[1] + this.offsetPoint[1] + j, this.pastePoint[2] + this.offsetPoint[2] + k);
+					};
```

Comment:
Putting this here one time for all classes; using Java 16 features are undesired at this time. Since Fawe still uses a minimum Java version of 11 and I see no good reason to require Java 16 here but for sake of using enhanced switches, 11 will remain.